test/README.md

```diff
@@ -1,72 +1,38 @@
-# Testing the WebSDK
+# Running WebSDK Test Suite
 
-## Testing Commands
-1. Run all and watch file changes: `yarn test`
-2. Run all with no watch: `yarn test:noWatch`
-3. Run specific file: `<one of above commands> <path to transpiled file>` e.g: `yarn test:noWatch build/ts-to-es6/test/unit/managers/ConfigManager.js`
+## VSCode Run a specific test
+_Tested up to VSCode Version: `1.54.2`_
+1. Install the ["Remote Development"](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) VS Code Extension
```

Comment:
More context might be useful here (think from the perspective of a newhire). Why are we running in the docker container? e.g: does this relate to transpilation or build process?